pkg/agent/route/route_windows.go

```diff
@@ -383,7 +394,7 @@ func (c *Client) syncRoute() error {
 	gwAutoconfRoute := &util.Route{
 		LinkIndex:         c.nodeConfig.GatewayConfig.LinkIndex,
 		DestinationSubnet: c.nodeConfig.PodIPv4CIDR,
-		GatewayAddress:    c.nodeConfig.GatewayConfig.IPv4,
+		GatewayAddress:    net.IPv4zero,
```

Comment:
Holder

Reply:
I think this should backport.

Reply:
It's introduced by https://github.com/antrea-io/antrea/commit/6143e78ed971f47ae8178b718b56dc25ca28e790

Reply:
You are right, I forgot that. Thanks for reminding.